db/migrate/20180810211514_add_voting_style_and_number_votes_per_heading_to_groups.rb

```diff
@@ -0,0 +1,6 @@
+class AddVotingStyleAndNumberVotesPerHeadingToGroups < ActiveRecord::Migration
+  def change
+    add_column :budget_groups, :voting_style, :string, default: 'knapsack'
```

Comment:
Good setup for `voting_styles`, opening possibilities for other voting mechanisms 👌

Reply:
For knapsack budgets that are currently in use, I think we would need a rake task to update the existing budgets 🤔, otherwise this field would still be nil.

Reply:
Sure, I will add a rake task.

Reply:
Apparently not...

I remember doing this some time back, but from recent I didn't as it seemed to do the trick. I had a dev seed db on v0.19 and then just ran the migration in this branch and it seems to have updated existing records.

```
consul_upstream=# select * from budget_groups where id = 1;
 id | budget_id | name |   slug   | max_votable_headings | voting_style | number_votes_per_heading 
----+-----------+------+----------+----------------------+--------------+--------------------------
  1 |         2 |      | all-city |                    1 | knapsack     |                        1
```

I already wrote the rake task, can include it in the PR if required.

Reply:
Oh that's right, because it's a new attribute 👍Right on, no need for an additional rake task. 🎉